packages/vite-plugin-svelte/src/utils/options.ts

```diff
@@ -363,6 +369,7 @@ export async function buildExtraViteConfig(
 		// Currently a placeholder as more information is needed after Vite config is resolved,
 		// the real Svelte plugin is added in `patchResolvedViteConfig()`
 		extraViteConfig.optimizeDeps.esbuildOptions = {
+			conditions: [...SVELTE_EXPORT_CONDITIONS],
```

Comment:
I don't think this is needed as Vite should handle the conditions too. Vite doesn't set the conditions as an esbuild option, but all path resolve will go through Vite's custom resolver so it's safe too.

Reply:
it didn't work without iirc but can test again. maybe vite itself should propagate custom conditions to esbuild from resolve.conditions?

Reply:
that would also make sense as a safety measure in Vite, but theoretically it shouldn't need to 🤔